repository/src/main/java/org/apache/atlas/repository/store/graph/v1/DeleteHandlerV1.java

```diff
@@ -1387,20 +1387,43 @@ private boolean skipClassificationTaskCreation(String classificationId) throws A
         try {
 
             List<String> taskTypes = Arrays.asList(CLASSIFICATION_REFRESH_PROPAGATION, CLASSIFICATION_PROPAGATION_DELETE);
-            if (RequestContext.get().getQueuedTasks().stream().anyMatch(task -> task.getClassificationId().equals(classificationId)
-                    && taskTypes.contains(task.getType()))) {
+            if (
+                    RequestContext.get().getQueuedTasks().stream()
```

Comment:
Add null check for "RequestContext.get().getQueuedTasks()"

Reply:
getQueuedTasks will always return empty array not null

Reply:
Added the check